content/webapp/components/InfoBox/InfoBox.tsx

```diff
@@ -23,36 +24,42 @@ const InfoContainer = styled(Space).attrs({
   background-color: ${props => props.theme.color('yellow')};
 `;
 
-const InfoIconWrapper = styled(Space).attrs({
+export const InfoIconWrapper = styled(Space).attrs({
   className: font('intb', 4),
   $h: { size: 's', properties: ['margin-right'] },
 })`
   float: left;
 `;
 
-const InfoBox: FunctionComponent<Props> = ({ title, items, children }) => {
+const InfoBox: FunctionComponent<Props> = ({
+  title,
+  items,
+  headingClasses = font('wb', 3),
+  children,
+}) => {
   return (
     <>
-      <h2 className={font('wb', 3)}>{title}</h2>
+      <h2 className={headingClasses}>{title}</h2>
       <InfoContainer>
-        {items.map(({ title, description, icon }, i) => (
-          <div key={i}>
-            {icon && (title || description) && (
-              <InfoIconWrapper>
-                <Icon icon={icon} />
-              </InfoIconWrapper>
-            )}
-            {title && <h3 className={font('intb', 5)}>{title}</h3>}
-            {description && (
-              <Space
-                $v={{ size: 'm', properties: ['margin-bottom'] }}
-                className={font('intr', 5)}
-              >
-                <PrismicHtmlBlock html={description} />
-              </Space>
-            )}
-          </div>
-        ))}
+        {items.length > 0 &&
```

Comment:
Is it worth extracting this condition above somewhere to make it clearer what's going on here? Relying on the right to left early exiting of the logical AND to execute the map might be confusing if someone isn't familiar.

Reply:
After talking to @rcantin-w, we can just remove `items.length` here (discussion of using the logical && below).